### PR TITLE
Fixed #2442 :When Recent Transactions is opened without connection of internet…

### LINKED ISSUE #2442: When Recent Transactions is opened without connection of internet App crashes
**Summary:**

When user is offline  if he opens Recent Transactions the App gets crashes instead it should show a Error layout of No Connection 

**Steps to reproduce:**

1.open mifos mobile app
2. turn off internet
3. open recent transactions

**Expected behaviour:**

It should show error layout of No Connection

**Observed behaviour:**

App crashes

**Screenshots:**

https://github.com/openMF/mifos-mobile/assets/118070186/e495debb-f380-4119-9b2e-a8fd3030ecab

### PR DESCRIPTION
**Pull request by kmanikanta335**
Fixes #2442 

Screenshots of UI changes.

https://github.com/openMF/mifos-mobile/assets/118070186/97667b8b-2cf0-44f8-89b6-48f550635820

Please make sure these boxes are checked before submitting your pull request - thanks!

- ✅  Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- ✅  Run the unit tests with `./gradlew check` to make sure you didn't break anything

- ✅  If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by kmanikanta335** [comment]
@PratyushSingh07 pls review it

**Comment by PratyushSingh07** [comment]
LGTM , we can merge this @therajanmaurya

**Comment by therajanmaurya** [comment]
Resolve the conflicts.

**Comment by ghost** [comment]
can i take solve this issue @PratayushSingh07

**Comment by PratyushSingh07** [comment]
> can i take solve this issue @PratayushSingh07

This pull request currently contains some merge conflicts. Once the author, @kmanikanta335 , resolves these conflicts, we'll proceed with merging it into our codebase

**Comment by kmanikanta335** [comment]
@therajanmaurya I resolved merge conflicts

File: app/src/main/java/org/mifos/mobile/ui/fragments/RecentTransactionsFragment.kt [hunk 1]
```diff
@@ -93,6 +93,7 @@ class RecentTransactionsFragment : BaseFragment(), OnRefreshListener {
 
                         RecentTransactionUiState.Initial -> {}
                     }
+
```
Comment (PratyushSingh07) [inline app/src/main/java/org/mifos/mobile/ui/fragments/RecentTransactionsFragment.kt]: this is the 1st blank space

File: app/src/main/java/org/mifos/mobile/viewModels/RecentTransactionViewModel.kt [hunk 2]
```diff
@@ -30,6 +30,7 @@ class RecentTransactionViewModel @Inject constructor(private val recentTransacti
 
     private fun loadRecentTransactions(offset: Int, limit: Int) {
         viewModelScope.launch {
+
```
Comment (PratyushSingh07) [inline app/src/main/java/org/mifos/mobile/viewModels/RecentTransactionViewModel.kt]: this is the 2nd blank space

**Review by PratyushSingh07 — CHANGES_REQUESTED** [review]
Hi @kmanikanta335 , I noticed you made some changes to the code by adding two new blank lines which is a bit weird because when you remove those 2 lines you will get the orignal code

**Pull request closed without merging**